articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -455,13 +455,10 @@ From the Azure portal, follow these steps:
 
```

Comment:
I can't add a comment on line 452. Can you please change "source App Service" to "target App Service resource"?